### PR TITLE
networkd: bridge convert to jiffies

### PR DESCRIPTION
**Pull request by ssahani**
fixes issue #1710

### CODE REVIEW

File: src/network/networkd-netdev-bridge.c [hunk 1]
```diff
@@ -72,20 +72,21 @@ static int netdev_bridge_post_create(NetDev *netdev, Link *link, sd_netlink_mess
         if (r < 0)
                 return log_netdev_error_errno(netdev, r, "Could not append IFLA_INFO_DATA attribute: %m");
 
+        /* convert to jiffes */
         if (b->forward_delay > 0) {
-                r = sd_netlink_message_append_u32(req, IFLA_BR_FORWARD_DELAY, b->forward_delay / USEC_PER_SEC);
+                r = sd_netlink_message_append_u32(req, IFLA_BR_FORWARD_DELAY, b->forward_delay / (sysconf(_SC_CLK_TCK) * 100));
```
Comment (zonque) [inline src/network/networkd-netdev-bridge.c]: I really think we shouldn't use `sysconf()` every time we do that division.  After all, we don't expect that value to change during runtime, do we?

Instead, jiffies/second or something similar should be determined once in a convenience function in `time-util.c`, and then stored in a `static` variable, so it is cached for subsequent calls.

Comment (poettering) [inline src/network/networkd-netdev-bridge.c]: yeah, I agree with @zonque.

Also, I think the actual conversion should be done in a generic usec_to_jiffies() call or so in time-util.c

**Comment by poettering** [comment]
Patch looks pretty good now, modulo the two issues I pointed out, but I didn't verify that the jiffies are correctly converted. I count that you tested and verified that this is now properly converted?

**Comment by ssahani** [comment]
yes I tested with this conf:

```
[NetDev]                                                                                                                                                                                      
Name=br-test                                                                                                                                                                                  
Kind=bridge                                                                                                                                                                                   

[Bridge]                                                                                                                                                                                      
ForwardDelaySec=4s                                                                                                                                                                            
```

$ brctl showstp br-test 

```
br-test
 bridge id      8000.82dbf39247fe
 designated root    8000.82dbf39247fe
 root port         0            path cost          0
 max age          20.00         bridge max age        20.00
 hello time        2.00         bridge hello time      2.00
 forward delay         4.00         bridge forward delay       4.00
 ageing time         300.00
 hello timer           0.00         tcn timer          0.00
 topology change timer     0.00         gc timer           0.00
 flags          

```

**Pull request closed without merging**